src/index.ts

```diff
@@ -8,20 +8,19 @@ import ENV from '../env.json';
 
 const app = express();
 
-const corsOptions = Object.freeze({
-    origin: ENV.CORS_ORIGIN,
-    optionsSuccessStatus: 200
-});
+const corsOptions = Object.freeze({});
```

Comment:
Is this empty object really needed? I suggest removing this and using `app.use(cors())` if we want to allow all CORS.

Reply:
You are right, removed